psi4/src/CMakeLists.txt

```diff
@@ -34,6 +34,13 @@ target_link_libraries(core PRIVATE ${psi4_binmodules})
 target_link_libraries(core PRIVATE ${LIBC_INTERJECT})
 target_link_libraries(core PUBLIC pybind11::module)
 target_link_libraries(core PRIVATE Threads::Threads)
+
+if(Fortran_ENABLED AND CMAKE_Fortran_COMPILER_ID MATCHES Intel)
```

Comment:
Ok, has my approval, though I might edit out the Intel later to avoid confusion.